src/renderer/components/Home.js

```diff
@@ -87,6 +92,7 @@ const StyledTabs = styled(Tabs)`
 const StyledTab = styled(Tab)`
   padding: 6px 24px 6px 18px;
   min-height: 64px;
+  max-width: 200px;
```

Comment:
This could lead to tab labels being truncated in different languages

Reply:
Do you have a suggested way to fix this that doesn't cause the sidebar to jump around?